### PR TITLE
doc: known issues: add Bluetooth LE entry

### PR DESCRIPTION
**Pull request by theob-pro**
Add DRGN-23511 and remove the statement that say that issues in "Bluetooth LE" are controller specific.

### CODE REVIEW

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 1]
```diff
@@ -268,6 +266,10 @@ DRGN-21390: The `disconnected` callback might not get called on nRF5340
 
   **Workaround:** Either disable host flow control (:kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL`) or cherry-pick commits from the upstream: `Zephyr PR #65272 <https://github.com/zephyrproject-rtos/zephyr/pull/65272>`_.
 
+DRGN-23511: Building of multilink Bluetooth applications fails if :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` is not set explicitly.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:269]:
```suggestion
.. rst-class:: v2-7-0

DRGN-23511: Building of multilink Bluetooth applications fails
  This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is not explicitly set.
```
Add any other affected versions to the first line.

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 2]
```diff
@@ -268,6 +266,10 @@ DRGN-21390: The `disconnected` callback might not get called on nRF5340
...
+
+  **Workaround:** Increase the Kconfig value accordingly.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:271]:
```suggestion
  **Workaround:** Set the Kconfig value explicitly.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 3]
```diff
@@ -268,6 +266,10 @@ DRGN-21390: The `disconnected` callback might not get called on nRF5340
 
   **Workaround:** Either disable host flow control (:kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL`) or cherry-pick commits from the upstream: `Zephyr PR #65272 <https://github.com/zephyrproject-rtos/zephyr/pull/65272>`_.
 
+.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1
+
+DRGN-23518: Reassembly broken when :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` is too small.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:271]:
```suggestion
DRGN-23518: Reassembly is sometimes broken
  This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is set to a value that is too small.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 4]
```diff
@@ -268,6 +266,11 @@ DRGN-21390: The `disconnected` callback might not get called on nRF5340
 
   **Workaround:** Either disable host flow control (:kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL`) or cherry-pick commits from the upstream: `Zephyr PR #65272 <https://github.com/zephyrproject-rtos/zephyr/pull/65272>`_.
 
+.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1
+
+DRGN-23518: Reassembly is sometimes broken
+  This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is set to a value that is too small.
```
Comment (laxiLang) [inline doc/nrf/releases_and_maturity/known_issues.rst:272]: what means "too small", can you rephrase it in a quantifiable description?

**Review by laxiLang — APPROVED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 5]
```diff
@@ -268,6 +266,12 @@ DRGN-21390: The `disconnected` callback might not get called on nRF5340
 
   **Workaround:** Either disable host flow control (:kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL`) or cherry-pick commits from the upstream: `Zephyr PR #65272 <https://github.com/zephyrproject-rtos/zephyr/pull/65272>`_.
 
+.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1
+
+DRGN-23518: ACL reassembly may deadlock if `CONFIG_BT_BUF_ACL_RX_COUNT <= CONFIG_BT_MAX_CONN`.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:271]:
```suggestion
DRGN-23518: ACL reassembly may deadlock
  This can happen if  the value of :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT` Kconfig option is less than or equal to the value of :kconfig:option:`CONFIG_BT_MAX_CONN`.
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 6]
```diff
@@ -268,6 +266,12 @@ DRGN-21390: The `disconnected` callback might not get called on nRF5340
...
+
+  **Workaround:** Ensure `CONFIG_BT_BUF_ACL_RX_COUNT > CONFIG_BT_MAX_CONN`.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:273]:
```suggestion
  **Workaround:** Ensure that the value of :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT` is greater than the value of :kconfig:option:`CONFIG_BT_MAX_CONN`.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]
